embedded/tbtree/tbtree.go

```diff
@@ -1434,7 +1481,7 @@ func (t *TBtree) IncreaseTs(ts uint64) error {
 	t.insertionCountSinceSync++
 
 	if t.insertionCountSinceFlush >= t.flushThld {
-		_, _, err := t.flushTree(false)
+		_, _, err := t.flushTree(0, false)
```

Comment:
Why it is not:

```suggestion
		_, _, err := t.flushTree(t.cleanupPercentage, false)
```

?